src/utils/source.js

```diff
@@ -375,6 +375,5 @@ export function getSourceClassnames(
   if (source.isBlackBoxed) {
     return "blackBox";
   }
-
-  return sourceTypes[getExtension(source)] || defaultClassName;
+  return sourceTypes[getExtension(source.url)] || defaultClassName;
```

Comment:
maybe name it getFileExtension or so for more clarity?

Reply:
yeah, that would be nice